lib/BackbeatConsumer.js

```diff
@@ -588,12 +670,16 @@ class BackbeatConsumer extends EventEmitter {
                 const doUnassign = () => {
                     try {
                         this._consumer.unassign();
+                        this._assignment = [];
                     } catch (e) {
                         // Ignore exceptions if we are not connected
                         const logger = this._consumer.isConnected() ? this._log.error : this._log.info;
                         logger.bind(this._log)('rdkafka.unassign failed', { e: e.toString(), assignment });
+                        return;
```

Comment:
for consistency, should we not resume in that case?

Reply:
Since we always resume when unassigning, i think this is not needed ?

Reply:
why do we need the `return` ?